README.md

```diff
@@ -237,6 +237,50 @@ inputView(_:shouldUpdatePosition:)
 ```
 
 
+## `CaseIterable` helpers
+
+A bunch of syntactic sugar to help with using `CaseIterable` enums to manage sections and rows in static tableviews. 
+
+This allows you to use something like
+
+```swift
+let item = Item.forRow(in: indexPath)
```

Comment:
not `at: indexPath`? 😅

Reply:
I'd be for `at` since its the same as the methods we all know from apple api ✌️